expr/parse.go

```diff
@@ -403,15 +403,15 @@ func parseString(s string) (string, string, error) {
 // metric name / path expression is defined by the following criteria:
 // 1. Not a function name
 // 2. Consists only of name characters
-// 	2.1 '=' is conditionally allowed if ';' is found (denoting tag format)
+// 	2.1 '=' / ' ' are conditionally allowed if ';' is found (denoting tag format)
```

Comment:
```suggestion
// 	2.1 '=' and ' ' are conditionally allowed if ';' is found (denoting tag format)
```
could be interpreted as saying that "/" is allowed in this case.